src/content/docs/go/docs/plugins/toolbox.md

```diff
@@ -0,0 +1,111 @@
+---
+title: MCP Toolbox for Databases
+---
+
+[MCP Toolbox for Databases](https://github.com/googleapis/genai-toolbox) is an open source MCP server for databases. It was designed with enterprise-grade and production-quality in mind. It enables you to develop tools easier, faster, and more securely by handling the complexities such as connection pooling, authentication, and more.
```

Comment:
We should add some guidance explaining why we should use this over the Genkit MCP plugin. I realized after digging there are extra features like Authenticated Parameters and Authorized invocation that aren't available in the MCP spec. Would be helpful to know that up front!